src/query/functions/src/scalars/array.rs

```diff
@@ -134,6 +137,76 @@ pub fn register(registry: &mut FunctionRegistry) {
         }))
     });
 
+    registry.register_passthrough_nullable_1_arg::<EmptyArrayType, EmptyArrayType, _, _>(
+        "array_sort",
+        FunctionProperty::default(),
+        |_| FunctionDomain::Full,
+        vectorize_1_arg::<EmptyArrayType, EmptyArrayType>(|arr, _| arr),
+    );
+
+    registry.register_passthrough_nullable_1_arg::<ArrayType<GenericType<0>>, ArrayType<GenericType<0>>, _, _>(
+        "array_sort",
+        FunctionProperty::default(),
+        |_| FunctionDomain::Full,
+        vectorize_1_arg::<ArrayType<GenericType<0>>, ArrayType<GenericType<0>>>(|arr, _| {
+            let data_type = arr.data_type();
```

Comment:
we can check the length of `arr`, if the number of elements is zero or one, don't need to be sorted.